### ISSUE TITLE
[openbmc]rflash support for firmware command - man page and usage

### ISSUE THREAD

**Issue by gurevichmark** [xcat2/xcat-core#3281]
Update man page and usage for openbmc `rflash` command.

**Comment by gurevichmark** [comment]
Include some sentences in man page stating that in the hierarchical environment the update file should be accessible to the SN, either using shared storage or manually copying the file to the SN from MN.

**Comment by zet809** [comment]
Hi, @gurevichmark , I also noticed that the inconsistent between the manpage and usage of rflash, will you pls also modify them? Thx!
* In manpage:
```
   OpenPOWER BMC specific (using IPMI):
       rflash noderange [hpm_file_path | -d=data_directory] [-c|--check] [--retry=count] [-V]

   OpenPOWER OpenBMC specific :
       rflash noderange [tar_file_path | image_id] [-c|--check] [-a|--activate] [-l|--list] [-u|--upload]
```
* But in usage:
```
    PPC64LE (using IPMI Management) specific:
        rflash <noderange> [-c|--check] [--retry=<count>] [-V] [<hpm_file>|-d=<data_directory>]
    PPC64LE (using OpenBMC Management) specific:
        rflash <noderange> [-c|--check] [-l|--list] [-a|--activate] [-u|--upload] [<tar_file>|<image_id>]
```

**Comment by gurevichmark** [comment]
@zet809 Are you concerned about the order in which options are listed ? Or different words used in the "xxxxx specific" line ?

**Comment by zet809** [comment]
Yes, the different words. It may confusing the user.

**Comment by gurevichmark** [comment]
Fixed by #3689
